website/content/docs/secrets/key-management/awskms.mdx

```diff
@@ -6,6 +6,9 @@ description: AWS KMS is a supported KMS provider of the Key Management secrets e
 
 # AWS KMS
 
+-> **Note**: This secret engine requires [Vault Enterprise](https://www.hashicorp.com/products/vault/)
+with the Advanced Data Protection Module.
```

Comment:
replace `with` to `for`?  Since the secret engine runs without VEw/ADPM, but ADPM doesn't run without VE?

Reply:
Nope, I'm wrong.